cpu/sam0_common/Makefile.features

```diff
@@ -4,6 +4,7 @@ FEATURES_PROVIDED += periph_flashpage_raw
 FEATURES_PROVIDED += periph_flashpage_rwee
 FEATURES_PROVIDED += periph_gpio periph_gpio_irq
 FEATURES_PROVIDED += periph_i2c_reconfigure
+FEATURES_PROVIDED += periph_spi_reconfigure
```

Comment:
The feature will need to be declared in `/kconfigs/Kconfig.features` and selected from `cpu/sam0_common/Kconfig`.

Reply:
Added.